libraries/AP_Logger/LogStructure.h

```diff
@@ -134,10 +134,10 @@ struct LogStructure {
 
 // maximum lengths of fields in LogStructure, including trailing nulls
 static const uint8_t LS_NAME_SIZE = 5;
-static const uint8_t LS_FORMAT_SIZE = 17;
+static const uint8_t LS_FORMAT_SIZE = 19;
 static const uint8_t LS_LABELS_SIZE = 65;
-static const uint8_t LS_UNITS_SIZE = 17;
-static const uint8_t LS_MULTIPLIERS_SIZE = 17;
+static const uint8_t LS_UNITS_SIZE = 19;
+static const uint8_t LS_MULTIPLIERS_SIZE = 19;
```

Comment:
Sorry, you can't do this.  `FMT` is essentially fixed.

Reply:
So there is no way to get this to work?